.gitignore

```diff
@@ -1,3 +1,6 @@
 .idea/
 *.pyc
 *.iml
+logging/
+config_dev.yaml
+passfile.txt
```

Comment:
Ok, understood. But I think it would be better to have a local configuration file which included your passwords / keys instead. Everything should be able to be put in the configuration file in this regard.